standalone/src/main.rs

```diff
@@ -241,7 +245,11 @@ fn main() {
                         vnic_name, 0x100, &hdl,
                     )?;
                     inv.register_instance(&viona, bdf.to_string())?;
-                    chipset.pci_attach(chipset.pci_root_bus(), bdf, viona);
+                    chipset.pci_attach(
+                        chipset.pci_root_bus(),
```

Comment:
If the bus in the parsed `bdf` is non-0, then the result of attaching to the root bus anyways (instead of throwing some sort of error) is surprising.

Reply:
Done.